src/components/ConsentPopup.js

```diff
@@ -0,0 +1,28 @@
+import { h } from 'preact'
+import { useI18n, useSoftkey } from 'hooks'
+import { setConsentStatus } from 'utils'
+
+export const ConsentPopup = ({ close }) => {
+  const i18n = useI18n()
+
+  const handleConsent = response => {
+    return () => {
+      setConsentStatus(response)
+      close()
+    }
+  }
+
+  useSoftkey('ConsentMessage', {
+    center: i18n('softkey-yes'),
+    onKeyCenter: handleConsent(true),
+    left: i18n('softkey-no'),
+    onKeyLeft: handleConsent(false)
```

Comment:
do we have the conclusion what should the backspace key action here?

Reply:
According to the phab descritption: 

> Pressing back on this dialog exists the app.

I tested this behavior on banana phone and app exists as expected.